apps/eth_blockchain/test/support/wait_for.ex

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule EthBlockchain.WaitFor do
+  @moduledoc """
+  Generic wait_for_* utils, styled after web3 counterparts
+  """
+
+  alias Ethereumex.HttpClient
+
+  def eth_rpc do
```

Comment:
```suggestion
  def eth_rpc(timeout \\ 10_000) do
```

For consistency with `eth_receipt/2`